engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -2157,6 +2157,24 @@ private IdentifiedLocation getIdentifiedLocation(Tree ast, boolean generateId) {
     return getIdentifiedLocation(ast, 0, 0, someId);
   }
 
+  private IdentifiedLocation getIdentifiedLocation(
+      Tree.Function fn, int b, int e, Option<UUID> someId) {
+    var orig = getIdentifiedLocation((Tree) fn, b, e, someId);
+    var start = castToInt(fn.getName().getStartCode()) + b;
```

Comment:
I am not really happy about the tricks that I have to do to remove comment from the location of `Tree.Function`, but unless @kazcw suggest easier way to do the same, I got with this code. (Multi line) comment just cannot be part of _executable_ `IdentifiedLocation` as also demonstrated by the [IGV problem](https://github.com/enso-org/enso/pull/14312#issuecomment-3562399308).